src/main/java/com/princexml/wrapper/AbstractPrince.java

```diff
@@ -679,6 +692,31 @@ public void setPdfOutputIntent(String pdfOutputIntent) {
         this.pdfOutputIntent = pdfOutputIntent;
     }
 
+    /**
+     * Include an AcroJS script to run when the PDF is opened.
+     * @param pdfScript The filename or URL of the AcroJS script.
```

Comment:
And `null` to clear?

Reply:
We never explicitly mention this in other setters that take a string. I'm fine with leaving it out; I assume someone using this library should figure it out quickly enough.

Reply:
Sounds fair!